### PR TITLE
Improve export functionality

### PR DESCRIPTION
**Pull request by tristanhcole**
**Describe the Pull Request**
Improves export functionality.

Adds the following columns:

- Amount received(integer): total amount received by the account
- Amount sen(integer)t: total amount sent by the account
- Amount available (Current Balance): amount received - amount sent
- Beneficiary(bool)
- Vendor(bool)

Screen Shot 2020-09-09 at 6 19 27 pm

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver

### CODE REVIEW

File: app/client/components/management/exportManager.jsx [hunk 1]
```diff
@@ -66,14 +66,6 @@ class ExportManager extends React.Component {
   }
```
Comment (tristanhcole) [inline app/client/components/management/exportManager.jsx:66]: this code is _oldddd_

Comment (tristanhcole) [reply app/client/components/management/exportManager.jsx:66]: needs a refactor at some point

**Comment by tristanhcole** [comment]
[Export Functionality](https://trello.com/c/bowexX4k/575-export-functionality)

**Review by enjeyw — APPROVED** [review]
Cool, merging on in!

**Pull request merged**